src/jit/codegenarm64.cpp

```diff
@@ -3692,6 +3692,996 @@ void CodeGen::genEmitHelperCall(unsigned helper, int argSize, emitAttr retSize,
     regTracker.rsTrashRegsForGCInterruptability();
 }
 
+#ifdef FEATURE_SIMD
+
+//------------------------------------------------------------------------
+// genSIMDIntrinsic: Generate code for a SIMD Intrinsic.  This is the main
+// routine which in turn calls apropriate genSIMDIntrinsicXXX() routine.
+//
+// Arguments:
+//    simdNode - The GT_SIMD node
+//
+// Return Value:
+//    None.
+//
+// Notes:
+//    Currently, we only recognize SIMDVector<float> and SIMDVector<int>, and
+//    a limited set of methods.
+//
+void CodeGen::genSIMDIntrinsic(GenTreeSIMD* simdNode)
```

Comment:
This function looks the same as the xarch one. Should we add a simdcodegencommon.cpp file and put this there (shared by all architectures, maybe with an `#ifdef` or two)?

Reply:
I'll do this in a separate PR